src/engraving/dom/pedal.cpp

```diff
@@ -201,8 +201,8 @@ PointF Pedal::linePos(Grip grip, System** sys) const
     } else {
         EngravingItem* e = endElement();
         ChordRest* c = toChordRest(endElement());
-        if (!e || e == startElement() || (endHookType() == HookType::HOOK_90)) {
-            // pedal marking on single note or ends with non-angled hook:
+        if (!e || e == startElement() || (endHookType() == HookType::HOOK_90) || !lineVisible()) {
```

Comment:
What do you think about changing this condition to 
```suggestion
        if (!e || e == startElement() || (endHookType() != HookType::HOOK_45)) {
```
?

Reply:
This makes sense